.github/workflows/ci.yml

```
@@ -84,12 +82,17 @@ jobs:
         with:
           platforms: all
 
-      - name: Build wheels
+      - name: Build wheels for Windows  # see issue #174
+        if: runner.os == 'Windows'
+        run: python -m cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_BUILD: cp38-${{ matrix.name }}* pp*-${{ matrix.name }}*
```

```suggestion
          CIBW_BUILD: cp38-${{ matrix.name }}*
```
OR `CIBW_PROJECT_REQUIRES_PYPY=1`